### ISSUE TITLE
Blank screen on iOS & Android with @ionic-native v4.11.0

### ISSUE THREAD

**Issue by diegozuber** [ionic-team/cordova-plugin-ionic#116]
As soon as I update all the `@ionic-native/{plugin_name}` to `v4.11.0` Ionic Deploy downloads the Update and then I just receive a blank white screen. Sometimes an endless splash screen on android. I'm getting the following error on the blank white screen in the console

iOS:
![bildschirmfoto 2018-08-02 um 16 33 10 pm](https://user-images.githubusercontent.com/21689209/43590768-14f36b12-9672-11e8-8a66-9a87bd69c95b.png)

Android:
![bildschirmfoto 2018-08-02 um 16 31 59 pm](https://user-images.githubusercontent.com/21689209/43590780-1ac40ce0-9672-11e8-9ba6-cbcfa702c38c.png)

I'm using:
```
cordova-plugin-ionic v5.0.5-rc.1
cordova-plugin-ionic-webview: v2.0.2
@ionic/pro v2.0.1
cordova-android v7.1.1
cordova-ios v4.5.5
```

EDIT: Currently I'm using these `@ionic-native` Version without any problem:
```
        "@ionic-native/barcode-scanner": "4.9.0",
        "@ionic-native/camera": "4.9.0",
        "@ionic-native/core": "4.9.0",
        "@ionic-native/in-app-browser": "4.9.0",
        "@ionic-native/keyboard": "4.9.0",
        "@ionic-native/onesignal": "4.9.0",
        "@ionic-native/screen-orientation": "4.9.0",
        "@ionic-native/splash-screen": "4.9.0",
        "@ionic-native/status-bar": "4.9.0",
```

**Comment by ambrosiajun** [comment]
hi @diegozuber I have same problem now. :) can you give me some advice for @angular/core version?

**Comment by diegozuber** [comment]
hey @ambrosiajun I'm using @angular/core version 5.0.3

Since I downgraded the Ionic Native Stuff back to 4.9.0 I had no problems with the Deployment.

**Comment by ambrosiajun** [comment]
thanks a lot i just open for github for someone like me.

https://github.com/ambrosiajun/ionic-pro-deploy-starter

**Comment by diegozuber** [comment]
@nphyatt Thanks, looks like it is working now. I will do some further testing.

**Comment by FrancisTse8** [comment]
How do I upgrade to 5.0.9? I did a _npm install --save @ionic-native/vibration@5.0.9_but got a npm ERR! notarget No matching version found for @ionic-native/vibration@5.0.9_. Any help is much appreciated!

**Comment by jcesarmobile** [comment]
He meant 5.0.9 of `cordova-plugin-ionic`, not `@ionic-native/vibration`. 
Latest is 5.2.5 now

**Comment by FrancisTse8** [comment]
Sorry to be dense about this. How am I supposed to upgrade to 5.0.9? Which plugin am I supposed to upgrade?

When I do a `cordova plugin list`, I did not see a `cordova-plugin-ionic` listed. This is what I have:
```
cordova-plugin-ble-central 1.1.4 "BLE"
cordova-plugin-compat 1.2.0 "Compat"
cordova-plugin-device 1.1.4 "Device"
cordova-plugin-ionic-webview 1.1.16 "cordova-plugin-ionic-webview"
cordova-plugin-splashscreen 4.0.3 "Splashscreen"
cordova-plugin-vibration 3.1.0 "Vibration"
cordova-plugin-whitelist 1.3.1 "Whitelist"
ionic-plugin-keyboard 2.2.1 "Keyboard"
```
Thought I am missing the `cordova-plugin-ionic`. And when I did a `cordova plugin add cordova-plugin-ionic`, I got the following:
```
Unmet project requirements for latest version of cordova-plugin-ionic:
    cordova-plugin-splashscreen (4.0.3 in project, >=5.0.1 required)
Fetching highest version of cordova-plugin-ionic that this project supports: 3.1.2 (latest is 5.2.5)
Fetching plugin "cordova-plugin-ionic@3.1.2" via npm
Error: Variable(s) missing (use: --variable APP_ID=value --variable CHANNEL_NAME=value).
```

**Comment by jcesarmobile** [comment]
You are commenting on `cordova-plugin-ionic` repo, so we are talking about `cordova-plugin-ionic`.

Latest version of `cordova-plugin-ionic` requires newer versions of `cordova-plugin-ionic-webview` and `cordova-plugin-splashscreen`.
You can remove those two and when you install `cordova-plugin-ionic` it will install both as those are dependencies.

`cordova-plugin-ionic` is a plugin for Ionic Pro, for some features like getting code updates and requires that your project is in Ionic Pro and you have to configure the plugin with a few variables that you'll get in the Ionic Pro dashboard.
Are you using Ionic Pro?

**Comment by FrancisTse8** [comment]
No, I am not using Ionic Pro.

**Comment by FrancisTse8** [comment]
Thanks for the clarification. Sorry, I was looking to solve a blank screen problem that happened when I added the cordova-plugin-vibration to a project that was working fine. Didn't realize what repo I was at when I thought I found the solution. This app was started as an Ionic 3 project and I've recently upgraded to Ionic 4. All else seems to be fine as I kept changing my code after the upgrade until I added the cordova-plugin-vibration. Don't know if my blank screen issue is related to the Ionic 4 upgrade. I will have to look into using Ionic Pro if it helps with code upgrade.
